README.md

```diff
@@ -230,7 +233,14 @@ vessels and divers) is fully utilised, starting by selecting randomly from the t
 then randomly from the priority reefs, and then - if there is spare capacity, which is
 rarely - randomly from the non-priority.
 
-### Depth Quality Control flags
+### Depth Data and Quality Control flags
+
+For calculation of pixel size depth raster data should have coordinates in unit metres, not
+degrees.
```

Comment:
Change this statement to be more general and move to the top, making clear that 

1. this set of scripts (attempts to) creates a standard set of geospatial data in a projection with a meter coordinate system
2. All functionality expects meter units

Reply:
I think the meter coordinate requirement only applies to the depth data. The output features etc should be in GDA2020 degrees.